nvflare/app_opt/tracking/mlflow/mlflow_writer.py

```diff
@@ -0,0 +1,106 @@
+# Copyright (c) 2023, NVIDIA CORPORATION.  All rights reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+from typing import Dict, Optional
+
+from nvflare.apis.analytix import AnalyticsDataType
+from nvflare.app_common.tracking.log_writer import LogWriter
+from nvflare.app_common.tracking.tracker_types import LogWriterName
```

Comment:
Since we are using the terms "tracking" everyone,
Does it make sense to have "LogWriter" -> "Tracker"
"LogWriterName" -> "TrackerName"?

Reply:
For this, I think it makes sense that LogWriter is a type of Tracker, so I think keeping this for now is okay, but we can change it if people think it makes more sense.